src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+input {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+}
+
+input.small {
+  background: url(images/Search.svg) no-repeat;
+  background-position: 12px 14px;
+  background-size: 11px 11px;
+}
+
+body {
+  margin-top: 0;
+}
+
+.searchbar {
+  margin-top: 20px;
+}
+
+.big {
+  padding: 0;
+  width: 100%;
+  height: 68px;
+  line-height: 21.86px;
+  color: rgba(61, 78, 97, 1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  text-indent: 62px;
+  font-family: Avenir Heavy, sans-serif;
+  font-size: 16px;
+  box-shadow: 0 1px 8px #3d4e611a;
+}
+
+.big::placeholder {
+  font-family: Avenir Book, sans-serif;
```

Comment:
don't change font-family for placeholder
set it for html and use
`font-family: inherit` on input styles
if you want to change weight, so use font-weight